tests/testit/test-revcheck.R

```diff
@@ -0,0 +1,13 @@
+library(testit)
+
+assert("check_deps() works", {
+  has_error(check_deps())
+
+  x <- check_deps("xfun")
+  (is.character(x$check))
+  (is.character(x$install))
+})
+
+assert("rev_check() needs a package name", {
+  has_error(rev_check())
```

Comment:
That's basically testing if a function can accept a missing argument without a default value, and we know for sure the answer is no.